docs/common/install_and_configure_nginx_with_rtmp_module_on_windows.md

```diff
@@ -40,6 +40,54 @@ events {
 }
 
 stream {
+    upstream secure-extraction-1 {
+        server 127.0.0.1:2951;
+    }
+    server {
```

Comment:
**nitpick(blocking)**: This configuration only has four RTMP/RTMPS endpoints for extraction, but we can extract up to 10 participants + screenshare. We must have 11 endpoints.